content/blog/greenlake-platform-infrastructure-monitoring-with-apache-skywalking.md

```diff
@@ -0,0 +1,144 @@
+
+SkyWalking could be used for monitoring and observability. In this blog post we would focus on how we could setup monitoring of infrastructure (Virtual Machine and/or Bare metal) instances using popular APM tool like SkyWalking.
+
+**Pre-requisite**
+
+* An active service subscription to HPE GreenLake Private Cloud Enterprise.
+* VM and/or Bare metal instances are provisioned for the workloads and we are able to login to console.
+* SkyWalking instance has been setup either on premises or in Hyperscalers. Ensure that inbound ingress traffic is allowed on port 11800 for SkyWalking instance.
+
+ **Steps to configure monitoring**
+
+1. Monitoring host metric with node exporter
+
+* Login to the console of machine to be monitored
+* Download and run the node exporter
+
+```
+wget https://github.com/prometheus/node_exporter/releases/download/v1.4.0-rc.0/node_exporter-1.4.0-rc.0.linux-amd64.tar.gz
+
+tar xvfz node_exporter-1.4.0-rc.0.linux-amd64.tar.gz
+
+rm -rf node_exporter-1.4.0-rc.0.linux-amd64.tar.gz
+
+cd node_exporter-1.4.0-rc.0.linux-amd64
+
+cp node_exporter /usr/sbin
+
+cat<<EOF >nodeexporter.service
+
+[Unit]
+Description=Service to start the node_exporter on the node
+[Service]
+Type=Simple
+ExecStart=/usr/sbin/node_exporter
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=%n
+
+[Install]
+WantedBy=multi-user.target
+
+EOF
+
+
+mv nodeexporter.service /etc/systemd/system
+systemctl daemon-reload
+systemctl enable nodexporter.service
+systemctl start nodexporter.service
+```
+
+ 
+
+*  Check the metrics endpoint shows metrics exported from the infrastructure node
+
+```
+curl <http://localhost:9100/metrics>
+```
+
+ 
+
+![](/img/node_metrics.png "Node Metrics")
+
+2. Setup OTel (OpenTelemetry) collector
+
+OTel provides a set of standardized vendor-agnostic SDKs, APIs, and tools for ingesting, transforming, and sending data to an Observability back-end. We can install OTel collector in the same host which exposes metrics or we can create central VM/Bare metal instance to receive telemetry information from several infrastructure instances.
+
+* Install OTel collector
+
+```
+sudo wget <https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.54.0/otelcol_0.54.0_linux_amd64.deb>
+
+sudo dpkg -i otelcol_0.54.0_linux_amd64.deb
+
+systemctl status otelcol
+```
+
+ 
+
+**Note:** By default, the otelcol systemd service will be started with the --config=/etc/otelcol/config.yaml option after installation.
+
+* Create a new otel configuration file (say /etc/otelcol/otel-collector-config.yaml) to link the metrics from host nodes to SkyWalking oap. The orange box in below images must be modified to point to correct infrastructure hosts and SkyWalking DNS or IP address.
+
+  
+
+![](/img/otel_collector_configuration.png "OTeL Configuration")
+
+* Change OTel configuration to newly created OTel configuration file
```

Comment:
If you decide you want to use text instead of bullets, what you can do is write out "Step 3: Validate the infrastucture..."

Also, take a look at this... did you really intend for one to be a bullet and one to be a numbered bullet?
![image](https://user-images.githubusercontent.com/77496261/207460167-332f145a-49aa-45f4-88cd-bf1739d563a0.png)